src/main/default/lwc/navigationButton/navigationButton.js

```diff
@@ -0,0 +1,89 @@
+import { LightningElement, api } from 'lwc';
+import next from '@salesforce/label/c.cookbook_Next';
+import back from '@salesforce/label/c.cookbook_Back';
```

Comment:
Please use `previous` instead of `back` so it is consistent with the `type` property.